api/peripheral_api/util/LedSign.js

```diff
@@ -0,0 +1,20 @@
+async function updateSignAxios(axios, req) {
+    await axios
+        .post('http://host.docker.internal:11000/api/update-sign' + {...req.body})
+        .then(() => {
+            return true
+        }).catch((err) => {
+            return false;
+        });
+}
+async function healthCheckAxios(axios) {
```

Comment:
why are you passing axios as a param? just import it at the library at the top
```suggestion
async function healthCheck() {
```